python/create-onchain-agent/templates/chatbot/pyproject.toml.jinja

```diff
@@ -11,7 +11,7 @@ python = "^3.10"
 python-dotenv = "^1.0.1"
 langchain-openai = "^0.2.4"
 langgraph = "^0.2.39"
-coinbase-agentkit = "0.1.2"
+coinbase-agentkit = "0.1.4"
```

Comment:
Let's use caret so the template can use the latest version, without us having to bump it
```suggestion
coinbase-agentkit = "^0.1.4"
```